gtd-timer/Controllers/LogInController.cs

```diff
@@ -0,0 +1,55 @@
+﻿using System.Threading.Tasks;
+using Microsoft.AspNetCore.Identity;
+using Microsoft.AspNetCore.Mvc;
+
+using gtdtimer.Services;
+using gtdtimer.Timer.DAL.Entities;
+using gtdtimer.Timer.DTO;
+
+namespace gtdtimer.Controllers
+{
+    [Route("api/[controller]")]
+    [ApiController]
+    public class LogInController : ControllerBase
+    {
+        private ApplicationUserManager<User> userManager;
+        private ILogInService logInService;
+        public LogInController(ApplicationUserManager<User> userManager, LogInService logInService)
+        {
+            this.userManager = userManager;
+            this.logInService = logInService;
+        }
+
+        [HttpPost]
+        [Route("login")]
+        public async Task<IActionResult> Login([FromBody] LoginDTO model)
+        {
+            if (model == null)
+            {
+                return BadRequest("Request is Null");
```

Comment:
sync with @YurasNazar and @nataliapachva

Reply:
I will create two classes exception that is using in CreateToken class later in the folder Exception